index.html

```diff
@@ -3410,6 +3410,194 @@ <h3>
+
+        <p>
+The output variables of this function MUST be as follows:
+        </p>
+
+        <dl>
+            <dt>
+did-url-dereferencing-metadata
+            </dt>
+            <dd>
+A <a href="metadata-structure">metadata structure</a> consisting of values
+relating to the results of the <a>DID URL Dereferencing</a> process. This
+structure is REQUIRED and in the case of an error in the dereferencing process,
+this MUST NOT be empty.
+Properties defined by this specification are in 
+<a href="#did-url-dereferencing-metadata-properties"></a>.
+If the dereferencing is not successful, this structure MUST contain an
+<code>error</code> property describing the error.
+            </dd>
+            <dt>
+content-stream
+            </dt>
+            <dd>
+If the <code>dereferencing</code> function was called and successful, this MUST
+contain a resource corresponding to the <a>DID URL</a>.
+The <code>content-stream</code> MAY be a <a>DID document</a> in one of the
+conformant <a href="#representations">representations</a> obtained through
+the resolution process.
+
+If the dereferencing is unsuccessful, this value MUST be empty.
+            </dd>
+            <dt>
+content-metadata
+            </dt>
+            <dd>
+If the dereferencing is successful, this MUST be a <a href="metadata-structure">
+metadata structure</a>, but the structure MAY be empty. This structure contains
+metadata about the <code>content-stream</code>.  
+If the <code>content-stream</code> is a <a>DID document</a>, this MUST be a
+<code>did-document-metadata</code> structure as described in <a>DID
+Resolution</a>.
+
+If the dereferencing is unsuccessful, this output MUST be an empty <a href="metadata-structure">metadata structure</a>.
+            </dd>
+        </dl>
+
+        <p>
+<a>DID URL Dereferencing</a> implementations MUST NOT alter the signature of
+these functions in any way. <a>DID URL Dereferencing</a> implementations MAY
+map the <code>dereference</code> function to a method-specific internal
+function to perform the actual <a>DID URL Dereferencing</a> process. <a>DID
+URL Dereferencing</a> implementations MAY implement and expose additional
+functions with different signatures in addition to the <code>dereference</code>
+function specified here.
+        </p>
+
+        <section>
+            <h3>
+DID URL Dereferencing Input Metadata Properties
+            </h3>
+
+            <p>
+The possible properties within this structure and their possible values are defined by [[DID-SPEC-REGISTRIES]].
+This specification defines the following common properties.
+            </p>
+
+            <dl>
+                <dt>
+accept
+                </dt>
+                <dd>
+The MIME type the caller prefers for <code>content-stream</code>. The <a>DID URL
+Dereferencing</a> implementation SHOULD use this value to determine the
+representation contained in the returned value if such a representation is
+supported and available. This property is OPTIONAL.
```

Comment:
wow... I don't think I've ever come across a spec with the OPTIONAL RFC keyword in it... we usually do "If provided..." -- keeping this in there for now... OPTIONAL doesn't result in test suite language.

Reply:
It's also in the (parallel) section of DID Resolution section: https://w3c.github.io/did-core/#did-resolution-input-metadata-properties